src/web/mod.rs

```diff
@@ -48,9 +48,8 @@ macro_rules! cexpect {
                 let request: &::iron::Request = $req;
 
                 ::log::error!(
-                    "called cexpect!() on a `None` value while attempting to fetch the route {:?}\n{:?}",
+                    "called cexpect!() on a `None` value while attempting to fetch the route {:?}",
                     request.url,
-                    ::backtrace::Backtrace::new(),
```

Comment:
Why remove this?

Reply:
it's the same backtrace as the one attached by sentry itself through `attach_stacktrace`.

Of course we could decide to keep it only for the logs on the server